media_select_FL.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
Napisal som to do PR andrasovi myslel som si ze to je jeho, aky zmysel maju skratky FL a FD vobec neviem co to znamena a to teraz na tom projekte pracujem. Napis tam pls nejake plnohodnotne nazvy